src/averageAges.js

```diff
@@ -37,10 +39,21 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const getPeople = people.filter(
```

Comment:
semantically these are `women`, right
`getPeople` is a good name for a function that would somehow process people array
```suggestion
  const women = people.filter(
```